src/Unleash/Internal/IToggleBootstrapProvider.cs

```diff
@@ -1,11 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Unleash.Internal
+﻿namespace Unleash.Internal
 {
     public interface IToggleBootstrapProvider
     {
-        ToggleCollection Read();
+        string Read();
```

Comment:
Breaking change

Reply:
Let's change the name of this one completely. Also now has a deprecation notice in main